src/main/java/io/prowave/chargify/webhook/bean/Dunner.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2016 Prowave Consulting, LLC
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.prowave.chargify.webhook.bean;
+
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Dunner {
+
+	@JsonProperty("subscription_id")
+	private String subscriptionId;
+
+	private String state;
+
+	private int attempts;
```

Comment:
Use the object wrappers like Integer for these primitives (Change in all classes) to prevent them from being included in JSON serialization as the primitive defaults.